### DISCUSSION TITLE
ANT MEDIA SERVER BUG INFINITY, HOW TO RESOLVE?

### DISCUSSION THREAD

**Discussion by KelvinVicctor** [ant-media/Ant-Media-Server#6728] [FAQs]
I'M USING THE ANT MEDIA SERVER BUT I'M NOT HAVING GOOD EXPERIENCES, MY BROADCATS IS LOW, AND MY CPU IS GOING AT 100% WITH JUST ONE LIVE, I HAVE THE PLAN OF 2 CPU, 4 GB RAM, 10 GB INTERNET, AND ALL THE LIVES THAT I PUT IT TO RUN ON YOUTUBE BEING VOD OR VIDEOS. M3U8 LOCKS THE SAME WAY, I POSSIBLY DIDN'T MAKE SOME CONFIGURATION, CAN ANYONE HELP ME?

NOTE: I'M FROM BRAZIL, I DON'T KNOW IF YOU INTERFERE

![baix](https://github.com/user-attachments/assets/be2c5131-28fd-4735-aefa-22fc1051cd5d)
![pesado](https://github.com/user-attachments/assets/372cf8f5-271c-4475-9eb2-0eaef65642cb)
![muito ruim](https://github.com/user-attachments/assets/32aee994-d573-4d94-bc23-faca887ccc19)

**Comment by amarantmeida** [comment]
@KelvinVicctor I will respond to you on the ticket as we would need access to your account to further debug the issue.

**Comment by KelvinVicctor** [comment]
haven't resolved it yet? I need to use the panel to do the lives, when will
I have it? Will it be resolved?

Em qui., 24 de out. de 2024 12:55, Amarnath Thodupunoori <
***@***.***> escreveu:

**Comment by lastpeony** [comment]
Disable ABR if its enabled through app settings.
